### PR TITLE
[RNMobile] Do not use hard coded `fontFamily` in Image block

### PR DESCRIPTION
**Pull request by daniloercoli**
This PR does fix an issue with the `Image` block on mobile where the `fontFamily` used was hard coded in the block, instead of reading it via CSS.

### CODE REVIEW

File: packages/block-library/src/image/styles.native.scss [hunk 1]
```diff
@@ -16,6 +20,10 @@
 	align-items: center;
 }
 
+.caption-text {
```
Comment (hypest) [inline packages/block-library/src/image/styles.native.scss]: Nit: for simplicity in accessing it, and to match the rest of the class definitions in the same file, what do you think about renaming it to `.captionText`?

**Review by hypest — APPROVED** [review]
This looks good to me!

I only left one comment that is not blocking; feel free to merge as is or do the renaming. Either way it's fine by me 👍

File: packages/block-library/src/image/styles.native.scss (hunk 1 above)
Comment (daniloercoli) [reply packages/block-library/src/image/styles.native.scss]: Yes, `captionText` was my 1st choice, but gets an error in JS lint if I rename it:
```
[0] gutenberg-mobile/gutenberg/packages/block-library/src/image/edit.native.js
[0]   294:57  error  ["captionText"] is better written in dot notation  dot-notation
```

Comment (hypest) [reply packages/block-library/src/image/styles.native.scss]: Aha, apparently the rules weren't in place in the past and the other class definitions had passed. Oh well, `caption-text` it is then :)

Comment (koke) [reply packages/block-library/src/image/styles.native.scss]: I think that message is not about the dash, but it's telling you to replace `styles['captionText']` with `styles.captionText`

**Pull request merged**